js/modules/k6/http/request_test.go

```diff
@@ -702,7 +704,7 @@ func TestRequestAndBatch(t *testing.T) {
 						stats.GetBufferedSamples(samples),
 						"GET",
 						sr("HTTPBIN_IP_URL/get"),
-						sr("HTTPBIN_IP_URL/redirect-to?url=HTTPSBIN_URL/set-cookie-and-redirect"),
+						"",
```

Comment:
Can we change to specifying the `name` as well here, see https://github.com/loadimpact/k6/pull/1570/files#r460821904 and the proposed changes below

Reply:
Resolved by a780923f.